### DISCUSSION TITLE
Bug fixes and Maven support

### DISCUSSION THREAD

**Discussion by afreakyelf** [afreakyelf/Pdf-Viewer#119] [General]
## What's Changed
* Fixed file not in PDF format or corrupted [#114](https://github.com/afreakyelf/Pdf-Viewer/issues/114), https://github.com/afreakyelf/Pdf-Viewer/issues/113
* Fixed scroll issue https://github.com/afreakyelf/Pdf-Viewer/issues/112
* Added `jumpToPage` to jump to a particular page https://github.com/afreakyelf/Pdf-Viewer/issues/111
* Fixed currentPage not closed issue https://github.com/afreakyelf/Pdf-Viewer/issues/109
* Added Maven central support.. Yay!! Painful but got it done! Thank you @vanniktech

**Full Changelog**: https://github.com/afreakyelf/Pdf-Viewer/compare/v2.0.7...v2.1.0

<hr /><em>This discussion was created from the release <a href='https://github.com/afreakyelf/Pdf-Viewer/releases/tag/v2.1.0'>Bug fixes and Maven support</a>.</em>

**Comment by shwetahatwar** [comment]
implementation "io.github.afreakyelf:Pdf-Viewer:2.1.1"
I have integrated this but still getting an error and my app crash 
java.lang.NoClassDefFoundError: Failed resolution of: Landroidx/compose/runtime/SnapshotFloatStateKt;

**Comment by shwetahatwar** [comment]
can you please help me with this

**Comment by dilip-jaiswal** [comment]
Hi Owner,

I have one concern regarding this pdf viewer library.
Why are you using READ_PHONE_STATE permission here ?

**Comment by vivek-np** [comment]
HI @afreakyelf  i am looking out for the solution for #126  can you provide any quick solution to this ? 
or do you have any plan to fix this issue

**Comment by afreakyelf** [comment]
Hey I have fixed above issue in this release https://github.com/afreakyelf/Pdf-Viewer/releases/tag/v2.2.0
